subsys/bluetooth/host/adv.c

```diff
@@ -1216,6 +1216,8 @@ static int le_ext_adv_param_set(struct bt_le_ext_adv *adv,
 
 	cp->sid = param->sid;
 
+	cp->sec_adv_max_skip = param->secondary_max_skip;
+
```

Comment:
Suggest to put this change into a separate commit

Reply:
Done